swift/Samples/SampleApp/Sources/CrossFadeContainer.swift

```diff
@@ -81,14 +79,14 @@ fileprivate final class CrossFadeContainerViewController: ScreenViewController<C
                 to: childViewController.view,
                 duration: 0.72,
                 options: .transitionCrossDissolve,
-                completion: { [childViewController] completed in
+                completion: { [childViewController] _ in
```

Comment:
IMO it can be helpful to see what is available to the block even if it isn't used

Reply:
Disabling this.